tasks/copy_to_glacier/README.md

```diff
@@ -80,39 +86,9 @@ in the cumulus-orca base of the repo.
 
 The `copy_to_glacier` lambda function can be deployed using terraform using the example shown in  `modules/lambdas/main.tf` file.
```

Comment:
Add that in the file to look at the resource "aws_lambda_function" "copy_to_glacier"

Reply:
changed to 
The `copy_to_glacier` lambda function can be deployed using terraform using the example shown in  `"aws_lambda_function" "copy_to_glacier"` resource block of `modules/lambdas/main.tf` file.